### PR TITLE
Allow to edit change amount

### PR DESCRIPTION
**Pull request by rohitwaghchaure**
Allow user to edit change amount from sales invoice form for is pos
Given provision to enter write off and change amount on POS

screen shot 2016-08-09 at 5 19 43 pm

### CODE REVIEW

**Comment by nabinhait** [comment]
@rohitwaghchaure Is it ready to merge?

**Comment by rohitwaghchaure** [comment]
Yes

File: erpnext/accounts/doctype/sales_invoice/test_sales_invoice.py [hunk 1]
```diff
@@ -455,6 +455,25 @@ def test_pos_gl_entry_with_perpetual_inventory(self):
 
 		self.pos_gl_entry(si, pos, 300)
 
+	def test_pos_chnage_amount(self):
```
Comment (nabinhait) [inline erpnext/accounts/doctype/sales_invoice/test_sales_invoice.py]: spelling?

File: erpnext/accounts/doctype/sales_invoice/sales_invoice.json [hunk 2]
```diff
@@ -2297,7 +2345,33 @@
...
+   "bold": 0, 
+   "collapsible": 0, 
+   "fieldname": "change_amount_account", 
+   "fieldtype": "Link", 
+   "hidden": 0, 
+   "ignore_user_permissions": 0, 
+   "ignore_xss_filter": 0, 
+   "in_filter": 0, 
+   "in_list_view": 0, 
+   "label": "Change Amount Account",
```
Comment (nabinhait) [inline erpnext/accounts/doctype/sales_invoice/sales_invoice.json]: does not sound good! May be "Account for Change Amount" is better.

**Pull request merged**